src/odemis/acq/align/roi_autofocus.py

```diff
@@ -119,22 +126,29 @@ def autofocus_in_roi(
         ccd: model.Component,
```

Comment:
Comment from another PR, Thera "autofocus_roi.py takes the current position of the focus component as the starting position, it might be useful to clarify this in the autofocus_roi docstring". -